src/easylink/configuration.py

```diff
@@ -147,6 +212,15 @@ def _get_schema(self, potential_schemas: list[PipelineSchema]) -> PipelineSchema
         exit_with_validation_error(dict(errors))
 
     def _validate(self) -> None:
+        """Validates the ``Config``.
+
+        If any errors are found, they are batch-logged into a dictionary and `exit()`
```

Comment:
fyi, I tried adding an "Exits" section but the font size was rendering poorly

```
Exits
-----
If any errors are found...
```

Reply:
maybe this needs to be in `raises`, `raise`, `except`, `exception` to work right?
https://www.sphinx-doc.org/en/master/usage/domains/python.html#info-field-lists